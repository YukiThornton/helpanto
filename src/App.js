import React, { Component } from 'react';
import './styles/App.css';
import VisibleRecipeList from './containers/VisibleRecipeList';
import RecipeDetailContainer from './containers/RecipeDetailContainer';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1 className="App-title">HELPANTO</h1>
          <p className="App-intro">
            Recipe keeper & cooking assistant
          </p>
        </header>
        <VisibleRecipeList />
        <RecipeDetailContainer />
      </div>
    );
  }
}

export default App;
