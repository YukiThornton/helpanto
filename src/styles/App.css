.App {
  text-align: center;
}

.App-header {
  background-color: #0097A7;
  height: 100px;
  padding: 20px;
  color: white;
}

.App-title {
  font-size: 1.5em;
  font-family: 'Exo', sans-serif;
}

.App-intro {
  font-size: large;
}

ul {
  list-style: none;
}

.Clickable {
  cursor: pointer;
}

.Spinner {
  height: 100px;
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
}
