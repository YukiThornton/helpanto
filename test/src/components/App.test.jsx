import React from 'react';
import { shallow } from 'enzyme';
import App from '../../../src/components/App';

describe('test App', () => {
  it('should render correctly', () => {
    const component = <App />;
    expect(shallow(component)).toMatchSnapshot();
  });
});
