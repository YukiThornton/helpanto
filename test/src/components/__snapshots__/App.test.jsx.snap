// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App should render correctly 1`] = `
<div
  className="App"
>
  <Connect(ErrorModal) />
  <Connect(NewRecipeModal) />
  <div
    className="App-visible"
  >
    <header
      className="App-header"
    >
      <h1
        className="Logo-font"
      >
        HELPANTO
      </h1>
      <p
        className="Subtitle"
      >
        Recipe keeper & cooking assistant
      </p>
    </header>
    <Connect(RecipeList) />
    <Connect(RecipeDetail) />
    <footer
      className="App-header"
      style={
        Object {
          "gridColumn": "1 / 3",
          "gridRow": "3 / 4",
        }
      }
    >
      <p
        className="Logo-font Subtitle"
      >
        HELPANTO
      </p>
    </footer>
  </div>
</div>
`;
